### PR TITLE
Added Stop Engaging After Focus

### PR DESCRIPTION
**Pull request by favour-chibueze**
Docs for https://github.com/mautic/mautic/pull/12742

### CODE REVIEW

File: docs/channels/focus_items.rst [hunk 1]
```diff
@@ -49,6 +49,7 @@ For each type of Focus Item, there are settings to configure which control the t
    - **Once per hour** -  Show the Focus Item once every hour that the visitor is on your website
    - **Once per day** - Show the Focus Item once per day that the visitor is on your website
 - **Stop engaging after a conversion** - This option is only available for the types which track a conversion (Collect Data and Emphasize a Link).  If set to Yes, the Focus Item no longer displays if the visitor has either submitted the Form (Collect Data type) or clicked on the link (Emphasize a Link type).
+- **Stop engaging after Focus** - This option is available for focus items and is based on cookies. If set to Yes, the focus item will no longer be displayed to the visitor after they have interacted with it, until they clear their cookies.
```
Comment (RCheesley) [inline docs/channels/focus_items.rst]:
```suggestion
- **Stop engaging after Focus** - This option is available for Focus items and uses cookies. If set to Yes, the Focus Item no longer displays to the visitor after they have interacted with it, until they clear their cookies.
```
I don't know why Vale is turned off entirely at Line 34 as it's not necessary (as far as I can see) and should have picked this up.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PR - some Vale fixes which were missed due to an earlier `vale off` statement!

File: docs/channels/focus_items.rst [hunk 2]
```diff
@@ -49,8 +47,7 @@ For each type of Focus Item, there are settings to configure which control the t
    - **Once per hour** -  Show the Focus Item once every hour that the visitor is on your website
    - **Once per day** - Show the Focus Item once per day that the visitor is on your website
 - **Stop engaging after a conversion** - This option is only available for the types which track a conversion (Collect Data and Emphasize a Link).  If set to Yes, the Focus Item no longer displays if the visitor has either submitted the Form (Collect Data type) or clicked on the link (Emphasize a Link type).
-
-.. vale on
+- **Stop engaging after Focus** - This option is available for Focus items and uses cookies. If set to Yes, the focus item no longer displays to the visitor after they have interacted with it, until they clear their cookies.
```
Comment (RCheesley) [inline docs/channels/focus_items.rst]:
```suggestion
- **Stop engaging after closing Focus** - This option is available for Focus Items and uses cookies. If set to Yes, the Focus Item no longer displays to the visitor after they have interacted with it, until they clear their cookies.
```

Sorry seems I missed this one!

**Review by RCheesley — CHANGES_REQUESTED** [review]

**Comment by patrykgruszka** [comment]
This is great, thank you @favour-chibueze!

**Review by RCheesley — APPROVED** [review]
LGTM! 🚀  (once the release is out!)

**Pull request merged**